internal/build/job/manager.go

```diff
@@ -37,6 +37,7 @@ func labels(mod kmmv1beta1.Module, targetKernel string) map[string]string {
 	return map[string]string{
 		constants.ModuleNameLabel:    mod.Name,
 		constants.TargetKernelTarget: targetKernel,
+		constants.JobType:            "build",
```

Comment:
Just because I am intrigued. How does it differ from https://github.com/kubernetes-sigs/kernel-module-management/blob/0c4df8dba30113323825ba1c12292880f88f6958/internal/build/job/maker.go#L126? Will it be easier to fetch jobs based on labels rather than on job's names?

Reply:
Seems like it is indeed easier to filter jobs based on labels cause it is supported by the client already.

Reply:
tldr: its easier because it's what we're already doing, and adding an extra label to match on is simpler than having to change the existing code :)

Longer answer: 
We only have the module name and the job-stage in GenerateName so we get the same GenerateName for different jobs building for different kernel versions. The jobs themselves are named  with GenerateName and a random 5 character string.

The job manager only manages a single job for a single `kernelmapping` Getting the job we want to manage would then need getting all the jobs that start with GenerateName and then somehow figuring out which one is the one we actually want to maange. Not impossible, but complicated and k8s has strong support for adding and searching by labels so I don't think searching by name would give us any advantages.

The only possible issue I can think of is you have two jobs running for the same module on the same kernel version when you'd have two jobs with the same labels returned, but they'd both have the same base name as well so a name search wouldn't improve this. (its an unlikely edge case that implies a problem anyway so which we currently handle getting multiple jobs back by throwing an error)